ci/openshift-ci/peer-pods-azure.sh

```diff
@@ -235,3 +235,15 @@ kubectl create ns default || true
 kubectl config set-context --current --namespace=default
 KATA_RUNTIME=kata-remote ./deploy_webhook.sh
 popd
+
+
+##################################
+# Log warning when peering created
+##################################
+if [[ "${AZURE_REGION}" != "${PP_REGION}" ]]; then
```

Comment:
I'm not familiar enough with the rest of the script, but at least from manual installation, I would expect that peering is somewhat distinct from the fact that we have different regions. In other words, different regions requires peering, but we might have later a need / desire to enable peering in the same region.

Would it be possible to add a variable that explicitly records when we activated peering? Probably more robust in the long run.

Reply:
Ack

Reply:
note the peering is usually not required and definitely not supported by kata. But currently it works(TM) and allows us to get coverage despite the region prow decides to run our jobs in. (upstream azure images are only available in eastus region)